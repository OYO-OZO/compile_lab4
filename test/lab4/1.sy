int a;

int main()
{
    int a,b;
    a = 1 + 2;
    b = 0x1f + 035;
    if(a < 5)
        return 1;
    return 0;
}