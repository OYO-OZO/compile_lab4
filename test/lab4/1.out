[DEBUG LAB4]: 	INT	int	1	0
[DEBUG LAB4]: 	ID	1	4	a	1
[DEBUG LAB4]: 	SEMICOLON	;	1	5
[DEBUG LAB4]: 	INT	int	3	0
[DEBUG LAB4]: 	ID	3	4	main	2
[DEBUG LAB4]: 	LP	(	3	8
[DEBUG LAB4]: 	RP	)	3	9
[DEBUG LAB4]: 	LC	{	4	0
[DEBUG LAB4]: 	INT	int	5	4
[DEBUG LAB4]: 	ID	5	8	a	3
[DEBUG LAB4]: 	COMMA	,	5	9
[DEBUG LAB4]: 	ID	5	10	b	4
[DEBUG LAB4]: 	SEMICOLON	;	5	11
[DEBUG LAB4]: 	ID	6	4	a	3
[DEBUG LAB4]: 	ASSIGN	=	6	6
[DEBUG LAB4]: 	NUMBER	}	6	8	1
[DEBUG LAB4]: 	ADD	+	6	10
[DEBUG LAB4]: 	NUMBER	}	6	12	2
[DEBUG LAB4]: 	SEMICOLON	;	6	13
[DEBUG LAB4]: 	ID	7	4	b	4
[DEBUG LAB4]: 	ASSIGN	=	7	6
[DEBUG LAB4]: 	NUMBER	7	8	31
[DEBUG LAB4]: 	ADD	+	7	13
[DEBUG LAB4]: 	NUMBER	7	15	NUMBER	29
[DEBUG LAB4]: 	SEMICOLON	;	7	18
[DEBUG LAB4]: 	IF	if	8	4
[DEBUG LAB4]: 	LP	(	8	6
[DEBUG LAB4]: 	ID	8	7	a	3
[DEBUG LAB4]: 	LESS	<	8	9
[DEBUG LAB4]: 	NUMBER	}	8	11	5
[DEBUG LAB4]: 	RP	)	8	12
[DEBUG LAB4]: 	RETURN	return	9	8
[DEBUG LAB4]: 	NUMBER	}	9	15	1
[DEBUG LAB4]: 	SEMICOLON	;	9	16
[DEBUG LAB4]: 	RETURN	return	10	4
[DEBUG LAB4]: 	NUMBER	}	10	11	0
[DEBUG LAB4]: 	SEMICOLON	;	10	12
[DEBUG LAB4]: 	RC	}	11	0
